### PR TITLE
docs: fix example code elastic_transcoder_preset

### PR DESCRIPTION
**Pull request by DavidJFelix**
The example elastic transcoder preset code is not consistent with a valid config. This change fixes the example code.

### CODE REVIEW

File: website/docs/r/elastic_transcoder_preset.html.markdown [hunk 1]
```diff
@@ -44,15 +44,15 @@ resource "aws_elastictranscoder_preset" "bar" {
     sizing_policy        = "Fit"
   }
 
-  video_codec_options = {
```
[the remark this answers is not in the document]
Comment (DavidJFelix) [reply website/docs/r/elastic_transcoder_preset.html.markdown]: Let me check this, because I tried it last night without equals and I think it worked. Let me try again and see what it wants and I'll update.

[the remark this answers is not in the document]
Comment (bflad) [reply website/docs/r/elastic_transcoder_preset.html.markdown]: It does work in Terraform 0.11, but in Terraform 0.12 (or sometime after) the validation might become stricter due to its enhanced support for typing (to distinguish between a configuration block and map). 😅

**Comment by bflad** [comment]
@DavidJFelix I believe the snake casing has been the pattern for Terraform attribute naming since its inception a few years ago (well before my time). We currently document lowercase snake case as best practice for Terraform provider development here as well: https://www.terraform.io/docs/extend/best-practices/naming.html#attribute-names

AWS API response values saved into Terraform attributes (or in the case of maps, both keys and values) should remain untouched for almost all resources and attributes. Generally speaking, we try to avoid any abstractions or response changes that may later become a maintenance burden for this project. It does wind up requiring some operational knowledge burden though for each service you interact with since AWS services do not have centralized API specifications or practices.
